examples/tutorials/earth-relief.py

```diff
@@ -95,8 +96,8 @@
 #
 # In addition to providing global data, the ``region`` argument for
 # :meth:`pygmt.datasets.load_earth_relief` can be used to provide data for a specific
-# area. The ``region`` argument is required for resolutions at 5 arc minutes or higher, and
-# accepts a list (as in the example below) or a string. The geographic ranges are
+# area. The ``region`` argument is required for resolutions at 5 arc minutes or higher,
+# and accepts a list (as in the example below) or a string. The geographic ranges are
 # passed as *xmin*/*xmax*/*ymin*/*ymax*.
 #
 # The example below uses data with a 5 arc minute resolution, and plots it on a
```

Comment:
For the regional map, could you change the resolution from 05m to 10m?

Reply:
Changed in f6f5f54